### DISCUSSION TITLE
Where to set go-runner image?

### DISCUSSION THREAD

**Discussion by Alan-Vlc** [litmuschaos/litmus#4317] [Q&A] [answered]
Good morning,

I have installed litmus-chaos (Version -2.15.11) on my kubernetes cluster. I have configured my local repository and everything starts correctly. However, when I launch an experiment and try to launch one of the pods that use the go-runner:2.14.0 image, it is not possible to download it because it does not take the configuration from my repository.

I have reviewed my values ​​file that I apply in my installation with helm but I don't see where I can specify it. I only see it possible to configure the following images

      imageEnv:
        SUBSCRIBER_IMAGE: "litmuschaos/litmusportal-subscriber:2.14.0"
        EVENT_TRACKER_IMAGE: "litmuschaos/litmusportal-event-tracker:2.14.0"
        ARGO_WORKFLOW_CONTROLLER_IMAGE: "litmuschaos/workflow-controller:v3.3.1"
        ARGO_WORKFLOW_EXECUTOR_IMAGE: "litmuschaos/argoexec:v3.3.1"
        LITMUS_CHAOS_OPERATOR_IMAGE: "litmuschaos/chaos-operator:2.14.0"
        LITMUS_CHAOS_RUNNER_IMAGE: "litmuschaos/chaos-runner:2.14.0"
        LITMUS_CHAOS_EXPORTER_IMAGE: "litmuschaos/chaos-exporter:2.14.0"

Is there a way to set the go-runner image repository?

The only option that I have tried and that works for me has been to modify the yaml of the pod that starts to establish my local repository

Thanks for your help...

I'm a noob

**Comment by namkyu1999** [comment]
Hi @Alan-Vlc , Here's a link: https://github.com/litmuschaos/litmus-go/tree/master

**Reply by Alan-Vlc** [reply]
Thanks for the reply. Sorry for my ignorance but I don't really understand where to look in that repository.

Maybe I haven't explained myself well, but I don't know how to specify in my values ​​file that I use with the helm installation the repository where to find the go-runner image. Taking into account that from my cluster I cannot access internet repositories.

**Reply by namkyu1999** [reply]
Our community welcome your question :)

**Reply by namkyu1999** [reply, marked as answer]
You can change the go-runner version in each Chaos Fault.
For example, If you want to change `go-runner` image version at `pod-delete `chaos fault, you need to change the values in [this file](https://github.com/litmuschaos/chaos-charts/blob/6d0fb711a2943294b68a70d93692fcdcc2a2b18d/faults/kubernetes/pod-delete/fault.yaml#L71).

**Reply by Alan-Vlc** [reply]
Thanks, I am aware that I can modify that configuration on the yaml.

However, I want to know if it would be possible to specify my repository during the installation process with helm. In this way, it would prevent users from having to make modifications every time they run a new experiment.

Thank you and excuse my insistence

**Reply by namkyu1999** [reply]
I recommend you fork the chaos-chart repository, change the version of the image, and register as a new chaoshub. IMO, the Unifying version of go-runner is not recommended because Each chaos fault does not guarantee they share the go-runner version.

**Comment by Alan-Vlc** [comment]
Good morning,
Thank you very much for your answers. I will take namkyu1999's proposal as a solution.

From my novice point of view, I think that this "need" to specify a repository for this type of images (example go-runner) should be parameterizable in some way.

Thank you all for your collaboration

Greetings
